package-managers/nuget.md

```diff
@@ -86,21 +94,67 @@ After making sure our feed is a public one we can consume it in Rider. To work w
 
 ### NuGet CLI
 
+NuGet package restore relies on the [NuGet.exe](https://nuget.org/nuget.exe) commandline tool by using the [install](https://docs.microsoft.com/nuget/tools/nuget-exe-cli-reference#Install_Command) command. The commandline will either prompt us for credentials or will look for credentials in the `NuGet.config` file in `%AppData%\NuGet\nuget.config` (if we use the Non-Interactive option).

```

Comment:
...`command <space> line tool` by using...

The NuGet command line client will prompt...

Reply:
What is the non-interactive option? Maybe: "if we use the `-NonInteractive` switch"